workspace.json

```diff
@@ -1525,6 +1525,35 @@
         }
       }
     },
+    "application-templates-data-protection-complaint": {
+      "root": "libs/application/templates/data-protection-complaint",
+      "sourceRoot": "libs/application/templates/data-protection-complaint/src",
+      "projectType": "library",
+      "schematics": {},
+      "architect": {
+        "lint": {
+          "builder": "@nrwl/linter:lint",
+          "options": {
+            "linter": "eslint",
+            "tsConfig": [
+              "libs/application/templates/data-protection-complaint/tsconfig.lib.json",
+              "libs/application/templates/data-protection-complaint/tsconfig.spec.json"
+            ],
+            "exclude": [
+              "**/node_modules/**",
+              "!libs/application/templates/data-protection-complaint/**/*"
+            ]
+          }
+        },
+        "test": {
+          "builder": "@nrwl/jest:jest",
+          "options": {
+            "jestConfig": "libs/application/templates/data-protection-complaint/jest.config.js",
+            "passWithNoTests": true
+          }
+        }
+      }
```

Comment:
You might want to add the extract-script to handle translation here https://docs.devland.is/handbook/libs/localization#message-extraction